standard/types.md

```diff
@@ -787,7 +787,7 @@ When the nullable context is ***annotations***:
 
 - For any reference type `T`, the annotation `?` in `T?` indicates that `T?` a nullable type, whereas the unannotated `T` is non-nullable.
 - No diagnostic warnings related to nullability are generated.
-- The null-forgiving operator `!` ([§12.8.9](expressions.md#1289-null-forgiving-expressions)) sets the null state of its operand to *not null*.
+- The null-forgiving operator `!` ([§12.8.9](expressions.md#1289-null-forgiving-expressions)) may alter the analyzed null state of its operand and what compile time informative messages are produced.
```

Comment:
We've used "diagnostic messages" elsewhere to discuss the nullability analysis. I think we should be consistent.

Reply:
@BillWagner – Thanks, I’ll go through and make sure they’re all the same.

Reply:
@BillWagner – A few moneys later… The Standard actually uses diagnostic to refer to both errors and warnings, as it is key the null analysis produces warnings I’ll go with *diagnostic warnings* (which is already used) everywhere.